README.md

```diff
@@ -48,6 +48,7 @@ See complete Ubuntu, Windows, and macOS templates in the [examples folder](https
 ### VM Location
 
 * `vm_name`(string) - Name of the new VM to create.
+* `annotation`(string) - Add some note.
```

Comment:
in UI it's named "Notes". We probably should follow end users, not internal APIs.